com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformTests.cs

```diff
@@ -42,45 +57,92 @@ public override IEnumerator Setup()
         }
 
         // TODO: rewrite after perms & authority changes
-        /*
-        [UnityTest]
-        [TestCase(true, NetworkAuthority.Client, ExpectedResult = null)]
-        [TestCase(true, NetworkAuthority.Server, ExpectedResult = null)]
-        [TestCase(false, NetworkAuthority.Client, ExpectedResult = null)]
-        [TestCase(false, NetworkAuthority.Server, ExpectedResult = null)]
-        public IEnumerator TestAuthoritativeTransformChangeOneAtATime(bool testLocalTransform, NetworkAuthority authorityToTest)
+        //*
```

Comment:
test was commented out, uncommenting, but still ignoring them. they pass in server mode, but not host mode. will need to investigate